### PR TITLE
feat: card component

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
image

image

image

### CODE REVIEW

File: pages/components/card.html [hunk 1]
```diff
@@ -106,8 +106,204 @@ <h1 clss="page-title">card</h1>
...
+          </figure>
+          <header class="card-header">
+            <h2>Title - Basic</h2>
+          </header>
+          <p class="card-body">
+            This hold the summary/ description
+          </p>
+        </section>
+        <section class="actions card-actions">
+          <section class="action-buttons">
```
Comment (manujagobind) [inline pages/components/card.html:124]: Can we use section within sections?

File: pages/components/card.html [hunk 2]
```diff
@@ -106,8 +106,204 @@ <h1 clss="page-title">card</h1>
...
+            <button class="btn bg-primary"><a href="/">Action 1</a></button>
+            <button class="btn outline-primary"><a href="/">Action 2</a></button>
+          </section>
+          <section class="action-icons">
+            <button class="btn btn-icon">
+              <span><img src="/assets/like.svg" alt="like" class="icon"></span>
+              Like
+            </button>
+
```
Comment (manujagobind) [inline pages/components/card.html:165]: HTML files can be formatted using prettier.

File: pages/components/card.html [hunk 3]
```diff
@@ -106,8 +106,204 @@ <h1 clss="page-title">card</h1>
...
+          <header class="card-header">
+            <h2>Title - Action Icons Only</h2>
+          </header>
+          <p class="card-body">
+            This hold the summary/ description
+          </p>
+        </section>
+        <section class="actions card-actions">
+          <section class="action-buttons">
+            <button class="btn bg-primary"><a href="/">Action 1</a></button>
```
Comment (manujagobind) [inline pages/components/card.html:255]: `/` in href will redirect to root page. Is this expected for this demo action?

File: styles/05-objects/card.css [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+.card-header {
+  margin: 1rem 0 0.5rem
+}
+
+.card-header,
+.card-body {
+  padding: 0 15px;
+}
+
+.decription {
```
Comment (manujagobind) [inline styles/05-objects/card.css]: Description is a generic name. If this is a generic text related selector, please move to typography.css

File: styles/05-objects/card.css [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+
+
+.actions button {
+  margin: 0 10px;
+  min-width: 50px;
+  min-height: 50px;
+  border: none;
+}
+
+
```
Comment (manujagobind) [inline styles/05-objects/card.css]: Remove extra line breaks.

File: styles/05-objects/card.css [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+
+
+.like {
```
Comment (manujagobind) [inline styles/05-objects/card.css]: Share/like should be part of social media css, not generic cards.css in my opinion. I could be wrong.

File: styles/05-objects/card.css [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+  position: relative;
+}
+
+.card-style-2 .card-header {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.card-style-3 .card-header,
```
Comment (manujagobind) [inline styles/05-objects/card.css]: Hard to remember what's style1/style2 etc. Would a text name make it easier?

File: styles/05-objects/card.css (hunk 4 above)
Comment (HarshnaNHaswani) [reply styles/05-objects/card.css]: Had renamed it, forgot to change in css. Thanks!

File: pages/components/card.html (hunk 1 above)
Comment (HarshnaNHaswani) [reply pages/components/card.html:124]: We can

File: pages/components/card.html (hunk 3 above)
Comment (HarshnaNHaswani) [reply pages/components/card.html:255]: no it is not

File: styles/05-objects/card.css (hunk 5 above)
Comment (HarshnaNHaswani) [reply styles/05-objects/card.css]: done!

File: styles/05-objects/card.css (hunk 7 above)
Comment (HarshnaNHaswani) [reply styles/05-objects/card.css]: Thanks for this. I changed them

File: styles/05-objects/card.css (hunk 6 above)
Comment (HarshnaNHaswani) [reply styles/05-objects/card.css]: not used, removed them.

File: pages/components/card.html (hunk 2 above)
Comment (HarshnaNHaswani) [reply pages/components/card.html:165]: already did

File: styles/05-objects/card.css [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+.cards {
+  padding: 10px;
+  justify-content: space-around;
+  gap: 4rem;
+}
+
+.card-container {
+  background-color: white;
```
Comment (mohitdhatrak) [inline styles/05-objects/card.css]: since you are using #fff below, keep i consistent use that here too

File: styles/05-objects/card.css [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+.card-title-top .card-header,
+.card-thumbnail .card-header {
+  order: -1;
+}
+
+.card-title-top .media img,
+.card-thumbnail .media img {
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
```
Comment (mohitdhatrak) [inline styles/05-objects/card.css]: maybe if this looks long, use the shorthand

File: styles/main.css [hunk 10]
```diff
@@ -19,6 +19,8 @@
 @import url(05-objects/alert.css);
 @import url(05-objects/badge.css);
 @import url(05-objects/button.css);
+@import url(05-objects/card.css);
+@import url(05-objects/icon.css);
```
Comment (mohitdhatrak) [inline styles/main.css:24]: why are there numbers in the url, is there a reason? also they are not in double quotes

**Review by mohitdhatrak — COMMENTED** [review]
1-2 small suggestions, good work!

File: styles/main.css (hunk 10 above)
Comment (HarshnaNHaswani) [reply styles/main.css:24]: Yes there is reason, its the name of the folder

File: styles/05-objects/card.css (hunk 8 above)
Comment (HarshnaNHaswani) [reply styles/05-objects/card.css]: Thanks for pointing it out

File: styles/05-objects/card.css (hunk 9 above)
Comment (HarshnaNHaswani) [reply styles/05-objects/card.css]: will do

**Comment by HarshnaNHaswani** [comment]
> 1-2 small suggestions, good work!

Thanks for your time and suggestion Mohit, will make the appropriate changes!

File: styles/05-objects/card.css (hunk 7 above)
Comment (HarshnaNHaswani) [reply styles/05-objects/card.css]: Thanks for your time and suggestions, I made the appropriate changes

**Pull request merged**